### PR TITLE
feat(web): person detail page

### PR DESCRIPTION
**Pull request by alextran1502**

### CODE REVIEW

**Comment by jrasm91** [comment]
Did you mean to open this against main?

**Comment by alextran1502** [comment]
> Did you mean to open this against main?

Nope, I forgot to change the base branch

File: server/apps/immich/src/controllers/person.controller.ts [hunk 1]
```diff
@@ -39,4 +40,13 @@ export class PersonController {
...
     return this.service.getAssets(id);
   }
+
+  @Put(':id')
+  update(
+    @GetAuthUser() authUser: AuthUserDto,
+    @Param() { id }: UUIDParamDto,
+    dto: UpdatePersonDto,
+  ): Promise<PersonResponseDto> {
+    return this.service.update(authUser.id, id, dto);
```
Comment (jrasm91) [inline server/apps/immich/src/controllers/person.controller.ts:50]: We should follow the pattern we use everywhere else and pass the entire auth user object.

File: server/libs/domain/src/people/person.service.ts [hunk 2]
```diff
@@ -13,6 +14,18 @@ export class PersonService {
     @Inject(IStorageRepository) private storageRepository: IStorageRepository,
   ) {}
 
+  async update(userId: string, personId: string, dto: UpdatePersonDto): Promise<PersonResponseDto> {
+    const person = await this.repository.getById(userId, personId);
+    if (!person) {
+      throw new NotFoundException();
```
Comment (jrasm91) [inline server/libs/domain/src/people/person.service.ts:20]: This should be a bad request exception.

**Review by jrasm91 — APPROVED** [review]

**Pull request merged**